lib/timeline/component/ItemSet.js

```diff
@@ -1456,8 +1458,8 @@ ItemSet.prototype._onDrag = function (event) {
         }
       }
 
-      var updateGroupAllowed = me.options.editable.updateGroup ||
-          props.item.editable === true;
+      var updateGroupAllowed = (me.options.editable.overrideItems && me.options.editable.updateGroup) ||
+          props.item.editable.updateGroup;
```

Comment:
What about: `!me.options.editable.overrideItems && me.options.editable.updateGroup`? You should be able to update groups but in this logic, you can't...